pkg/pods/manifest.go

```diff
@@ -123,6 +128,13 @@ func (manifest *Manifest) Bytes() ([]byte, error) {
 	return yaml.Marshal(manifest)
 }
 
+// Returns bytes correlating to the (potentially mutated) struct, unlike
+// Bytes() which guarantees that the bytes returned will be the same bytes the
+// manifest struct was built from
+func (manifest *Manifest) Marshal() ([]byte, error) {
```

Comment:
I realize that callers can just call `yaml.Marshal()` themselves, but I think this will make the codebase clearer